optional_plugins/html/examples/tests/passtest.py

```diff
@@ -0,0 +1,15 @@
+from avocado import Test
```

Comment:
Why some plugins has tests inside examples and others don't? Probably I'm missing something here.

Reply:
> Why some plugins has tests inside examples and others don't? Probably I'm missing something here.

Because some plugins are using these files for the examples while the others don't. 
Going further, some plugins don't even have any test (varianter_pict, resultsdb and result_upload)